test/IgnoreOrder.test.js

```diff
@@ -0,0 +1,44 @@
+import path from 'path';
+import webpack from 'webpack';
+
+describe('IgnoreOrder', () => {
+  it('should emit errors', (done) => {
+    const casesDirectory = path.resolve(__dirname, 'cases');
+    const directoryForCase = path.resolve(casesDirectory, 'ignoreOrderFalse');
+    // eslint-disable-next-line import/no-dynamic-require, global-require
+    const webpackConfig = require(path.resolve(
+      directoryForCase,
+      'webpack.config.js'
+    ));
+    const compiler = webpack({
+      ...webpackConfig,
+      mode: 'development',
+      context: directoryForCase,
+      cache: false,
+    });
+    compiler.run((err1, stats) => {
+      expect(stats.hasWarnings()).toBeTruthy();
```

Comment:
`toBe(true)`

Reply:
just wondering why toBeTruthy(); is "better". Seems like the exact same shit to me...

Reply:
@evilebottnawi